src/components/home/Connected.jsx

```diff
@@ -22,6 +22,26 @@ const Connected = () => {
           />
         ))}
       </div>
+      <div className=" grid grid-cols-3 mt-[3%] mb-[3%] w-3/4 sm:hidden ">
```

Comment:
you can use a single map without using slice.

try using flex wrap and center it instead.

https://tailwindcss.com/docs/flex-wrap